### ISSUE TITLE
androidiff.py AttributeError: 'module' object has no attribute 'SIGNATURE_L0_4'

### ISSUE THREAD

**Issue by xzhou** [androguard/androguard#157]
Traceback (most recent call last):
  File "./androdiff.py", line 32, in <module>
    from elsim.elsim_dalvik import ProxyDalvik, FILTERS_DALVIK_SIM, ProxyDalvikMethod, FILTERS_DALVIK_BB
  File "./elsim/elsim/elsim_dalvik.py", line 28, in <module>
    DEFAULT_SIGNATURE = analysis.SIGNATURE_L0_4
AttributeError: 'module' object has no attribute 'SIGNATURE_L0_4'

**Comment by amalsajeevan** [comment]
I am also facing the same issue while running ./androdiff.py 
Below i am mentioning the error  :

File "./androdiff.py", line 32, in <module>
    from elsim.elsim_dalvik import ProxyDalvik, FILTERS_DALVIK_SIM, ProxyDalvikMethod, FILTERS_DALVIK_BB
  File "./elsim/elsim/elsim_dalvik.py", line 28, in <module>
    DEFAULT_SIGNATURE = analysis.SIGNATURE_L0_4
AttributeError: 'module' object has no attribute 'SIGNATURE_L0_4'

Can you please help me to resolve this issue.
Thankyou

**Comment by lingfennan** [comment]
I have this bug as well. It seems some new commits break the dependency. I looked back and found this commit (cd0f6a60a639cdecc4df40948fb0a172657f8b3b) works.
`git checkout cd0f6a60a639cdecc4df40948fb0a172657f8b3b`
BTW, the programs depends on relative path of elsim, i.e. "./elsim", I did a sed to make it absolute path, so that I can use androsim.py at other locations.
`sed -i 's/.\/elsim/$ESCAPED_ABS_PATH/g' *.py`
